lib/css/modules/_userTagger.scss

```diff
@@ -3,6 +3,7 @@
 	position: absolute;
 	width: 336px;
 	box-sizing: border-box;
+	z-index: 35;
```

Comment:
There's a z-indexes.scss with lots of variables, can you use a variable from there? Maybe add one if needed.